example/android/app/src/main/kotlin/com/jhomlala/better_player_example/BetterPlayerNotificationService.kt

```diff
@@ -0,0 +1,183 @@
+package com.jhomlala.better_player_example
+
+import android.annotation.SuppressLint
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.app.PendingIntent
+import android.app.Service
+import android.content.Context
+import android.content.Intent
+import android.graphics.Bitmap
+import android.graphics.drawable.Drawable
+import android.os.Build
+import android.os.IBinder
+import android.support.v4.media.session.MediaSessionCompat
+import android.util.Log
+import androidx.annotation.RequiresApi
+import androidx.core.app.NotificationCompat
+import androidx.lifecycle.Observer
+import com.google.android.exoplayer2.ui.R
+import com.jhomlala.better_player.BetterPlayerPlugin
+import com.squareup.picasso.Picasso
+import com.squareup.picasso.Target
+
+class BetterPlayerNotificationService: Service() {
```

Comment:
Change the name from `BetterPlayerService` to be more readable.
To show Notification, and to manage button in notification.